Library/Homebrew/test/dev-cmd/pull_spec.rb

```diff
@@ -10,10 +10,13 @@
       .and not_to_output.to_stdout
       .and be_a_failure
 
-    expect { brew "pull", "1" }
-      .to output(/Fetching patch/).to_stdout
-      .and output(/Current branch is new\-branch/).to_stderr
-      .and be_a_failure
+    # Needs Homebrew/homebrew-core
+    if OS.mac?
+      expect { brew "pull", "1" }
+        .to output(/Fetching patch/).to_stdout
+        .and output(/Current branch is new\-branch/).to_stderr
+        .and be_a_failure
+    end
```

Comment:
If this test is passing now using Linuxbrew/homebrew-core, it should probably pass once again once that repo is renamed and moved to Homebrew/linuxbrew-core. If so, this chunk can be reverted after this transfer is complete.

Reply:
It didn't, unfortunately, but I'm not convinced that's a problem/priority.

Reply:
I think it's because https://github.com/Homebrew/homebrew-core/pull/1 is a pull request but https://github.com/Homebrew/linuxbrew-core/issues/1 is an issue

Reply:
running a fix through CI in https://github.com/Homebrew/brew/pull/5808